2023/AzureCGN/data.json

```diff
@@ -0,0 +1,26 @@
+{
+  "ValidMaintainers": {
+    "Description": "GitHub users who maintain this folder. PR for changing this folder will only be accepted from these users.",
+    "GitHubHandles": [
+      "rakoellner", "PfiffiKrebs","christianwaha"
+    ]
+  },
+  "CommunityInfo": {
+    "Name": "Azure Meetup Cologne",
+    "Country": "Germany",
+    "Url": "https://www.rakoellner.de/global-azure-cgn/",
+    "SessionizeApiId": "",
+    "Logo": {
+      "Description": "Community logo is a .png file 500X500px named {foldername}.png.",
+      "File": "_azurecgn-logo.png"
```

Comment:
Typo here in the file name, right? '_'

Reply:
*ping* @christianwaha!

Reply:
Hi Magnus, yes seems that there is an underscore, which should be not there.